site/templates/nav.html

```diff
@@ -12,6 +12,7 @@
             <li $if(isCommunity)$     class="active" $endif$ ><a href="/community/">Community</a></li>
             <li $if(isDocumentation)$ class="active" $endif$ ><a href="/documentation/">Documentation</a></li>
             <li $if(isDonations)$     class="active" $endif$ ><a href="/donations/">Donate</a></li>
+            <li $if(isSetUp)$         class="active" $endif$ ><a href="/start-coding/">Start Coding now</a></li>
```

Comment:
This should be the first link in the list.

Reply:
Done